src/world.ts

```diff
@@ -300,4 +312,16 @@ export class World {
         this._recordJoin({ name, ip })
         this._events.onJoin.dispatch(this.getPlayer(name))
     }
+
+    /**
+     * Sends messages in the message queue one by one to prevent the api from concatting 2 messages into 1.
+     */
+    protected async _executeMessageQueue () : Promise<void> {
+        while (this._messageQueue.length > 0) {
+            const {message, callback} = this._messageQueue[0]
+            await this._api.send(message)
+            callback()
```

Comment:
If the API ever throws, this may cause the `send` function to stop working completely, instead, catch the error. (Don't log for now, I'm still trying to figure out how this library should ideally report errors)